frontend/packages/container-security/src/components/summary.tsx

```diff
@@ -44,14 +44,25 @@ export const quayURLFor = (vuln: ImageManifestVuln) => {
 
 export const SecurityBreakdownPopup: React.FC<SecurityBreakdownPopupProps> = ({
   imageManifestVuln,
+  namespace,
 }) => {
   const resource = imageManifestVuln.data;
-
   const vulnsFor = (severity: string) =>
     resource.filter((v) => v.status?.highestSeverity === severity);
   const fixableVulns = resource
     .filter((v) => v.status?.fixableCount > 0)
     .reduce((all, v) => all.set(v.metadata.name, v), new Map<string, ImageManifestVuln>());
+  const getVulnerabilityCountText = (v: ImageManifestVuln, n?: string): string => {
+    const { highCount, mediumCount, lowCount, unknownCount } = v.status;
```

Comment:
can `v.status` be undefined?
```suggestion
    const { highCount =0, mediumCount=0, lowCount=0, unknownCount=0} = v.status || {};
```

Reply:
`v.status` shouldn't be undefined as it is a required property in the type `ImageManifestVuln`